test/Microsoft.TestPlatform.AcceptanceTests/RunsettingsTests.cs

```diff
@@ -8,8 +8,6 @@
 using Microsoft.TestPlatform.TestUtilities;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-#nullable disable
```

Comment:
Changes to this file are unrelated to this PR but fixes the failing acceptance tests that were blocking this PR.